app/stac_api/managers.py

```diff
@@ -215,12 +215,16 @@ def filter_by_query(self, query):
             for operator in query[attribute]:
                 value = query[attribute][operator]  # get the values given by the operator
 
+                if attribute in ["updated", "created"]:
+                    prefix = ""
+                else:
+                    prefix = "properties_"
+
                 # __eq does not exist, but = does it as well
                 if operator == 'eq':
-                    query_filter = f"properties_{attribute}"
+                    query_filter = f"{prefix}{attribute}"
                 else:
-                    query_filter = f"properties_{attribute}__{operator.lower()}"
-
+                    query_filter = f"{prefix}{attribute}__{operator.lower()}"
                 return self.filter(**{query_filter: value})
```

Comment:
just an idea: would it make code simpler by removing the `properties_` prefix in the model from the other parameters? then no special treatment would be necessary here

Reply:
Probably it would, yes. A global search for "properties" in the code yields more than 600 hits, so it might take some time to figure out, where it needs to be replaced and what influences that has, though.

Reply:
I see ...